code/modules/paperwork/paperbin.dm

```diff
@@ -52,7 +53,7 @@
 		var/obj/screen/inventory/hand/H = over_object
 		M.putItemFromInventoryInHandIfPossible(src, H.held_index)
 
-	add_fingerprint(M)
+	src.add_fingerprint(M)

```

Comment:
bad change

Reply:
I noticed the fingerprint wasn't getting added without "src.". But this could've been due to lag as I noticed later that adding fingerprints has a lot of delay.

Reply:
this would not be the cause, the statements are 100% equivalent. Except the src version goes against code guidelines

Reply:
Alright, I will remove it.